app/code/Magento/Checkout/Controller/Cart.php

```diff
@@ -53,13 +59,15 @@ public function __construct(
         \Magento\Checkout\Model\Session $checkoutSession,
         \Magento\Store\Model\StoreManagerInterface $storeManager,
         \Magento\Framework\Data\Form\FormKey\Validator $formKeyValidator,
-        CustomerCart $cart
+        CustomerCart $cart,
+        Escaper $escaper
```

Comment:
Hi @rafaelstz Please, review this document: http://devdocs.magento.com/guides/v2.0/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter

There is an example how to add new constructor argument in a backward compatible way (mentioned by @ihor-sviziev)